proto/beacon/rpc/v1/services.proto

```diff
@@ -179,12 +179,13 @@ message BlockTreeResponse {
 
 enum ValidatorStatus {
   UNKNOWN_STATUS = 0;
-  PENDING_ACTIVE = 1;
-  ACTIVE = 2;
-  INITIATED_EXIT = 3;
-  WITHDRAWABLE = 4;
-  EXITED = 5;
-  EXITED_SLASHED = 6;
+  DEPOSIT_RECEIVED = 1;
```

Comment:
What's the difference between deposit received and pending active?

Reply:
Deposit received means that we have observed the deposit log for that validator but the deposit isnt processed into the chain yet.